docs(GetPathwayDataPointDefinitions): fix typo, add comments and clean up (#129)
awell-health/developer-hub: content/awell-orchestration/api-reference/queries/get-data-point-definitions.mdx

    release_id: $release_id
  ) {
    data_point_definitions {
      id # unique identifier
      title # string
      key # string
      category # CALCULATION, FORM, PATIENT_PROFILE, PATHWAY, TRACK, STEP
      valueType # STRING, NUMBER, DATE, BOOLEAN, NUMBERS_ARRAY
      possibleValues { # only for STRING value type
        value # string
        label # string
      }
      range {
        min # number
        max # number
      }
      unit # string
      optional # boolean
      pii # boolean
      data_point_metadata { # list of key-value objects
        key # string

>         value # string

Reviewer: @rahulkeerthi I am not too sure about all of the comments we place after the fields. It makes the query looks a bit messy and hard to read, and when I want to have a "clean" copy then I have to delete all the comments manually. 

Additionally, GraphQL is a typed query language so the added benefit of manually highlighting the types seems limited. They can always have a look at the schema to understand the types.

I do like the additional paragraph on data point metadata.

      }
    }
  }
}
```

> `data_point_metadata` is a list of key-value pairs (`{ "key": "my_key", "value": "my_value"}`) that can be used to store additional information about a data point definition. This is configured in the Data Catalog section of Studio, and a guide can be found in [the Awell knowledge base](https://help.awellhealth.com/en/articles/6619338-configure-use-your-data-catalog).

Reviewer: ```suggestion
The `data_point_metadata` field can be used to retrieve additional information about a data point definition. This additional information can be configured in the Data Catalog section of Awell Studio, and a guide can be found in [the Awell knowledge base](https://help.awellhealth.com/en/articles/6619338-configure-use-your-data-catalog).
```
